clowdapp.yaml

```diff
@@ -28,6 +28,26 @@ objects:
           limits:
             cpu: ${CPU_LIMIT}
             memory: ${MEMORY_LIMIT}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
```

Comment:
Here it should be `/metrics` and not `/`. Ref - https://github.com/RedHatInsights/ros-ocp-backend/blob/main/internal/utils/utils.go#L183

Reply:
Thanks @patilsuraj767, change pushed, do the the config params look fine?

Reply:
Config look good for now, Later if we think that there are many http requests coming to the pod we can increase the `periodSeconds` time.